blockbuilder/genesisState.go

```diff
@@ -0,0 +1,33 @@
+package blockbuilder
+
+import (
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/validator"
+)
+
+type GenesisConfig struct {
+	ChainID                 string           `json:"chain_id" validate:"required"`
+	WhitelistedSequencerSet []felt.Felt      `json:"whitelisted_sequencer_set"`
```

Comment:
I think it is fine to get rid of this, MVP will include a single centralized sequencer.